### PR TITLE
Picture Slides Lab should stay inside screen at low resolution #1017

### LINKED ISSUE #1017: Picture Slides Lab should stay inside screen at low resolution
When the resolution is low (e.g. forced lower resolution when connected to projector), PSL window goes out of screen and PSL becomes unusable.

### PR DESCRIPTION
**Pull request by Dawnlord**
Fixes #1017 
 It can fit the screen even 800 * 600.

It is a picture of the screen resolution of 800*600.

(BTW sorry the wrong word in the commit)

### CODE REVIEW

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml [hunk 1]
```diff
@@ -10,8 +10,8 @@
              WindowStartupLocation="CenterScreen"
              Name="Window"
              Title="Picture Slides Lab"
-             Width="1200"
-             Height="700"
+             
+                           
              WindowTitleBrush="#0070C0"
```
Comment (Dawnlord) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml]: Only these 4 lines are changed by me. The others are auto changed by the Visual Studio.

Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml]: pls remove unnecessary changes

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs [hunk 2]
```diff
@@ -66,6 +66,11 @@ public partial class PictureSlidesLabWindow : IPictureSlidesLabWindowView
         private bool _isDisplayDefaultPicture;
         private bool _isEnableUpdatePreview = true;
 
+        private int sysHeight;
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs]: naming
1. better start with underscore `_` to indicate it's a private member.
2. better use full word such as `system`, `height`, `width` instead of `sys`, `H`, `W`

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs [hunk 3]
```diff
@@ -88,6 +93,9 @@ private void Init()
         {
             try
             {
+                GetScreenSize();
+                InitSize();
+                InitPosition();
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs]: can all these methods be merged into one?
and avoid those additional private members?

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs [hunk 4]
```diff
@@ -105,6 +113,29 @@ private void Init()
             }
         }
 
+        private void GetScreenSize()
+        {
+            System.Drawing.Size mSize = SystemInformation.WorkingArea.Size;
+            sysHeight = mSize.Height;
+            sysWidth = mSize.Width;
+            sizeW = sysWidth * 1.0 / 1280;
+            sizeH = sysHeight * 1.0 / 800;
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs]: why it's against 1280 x 800? could it be other figures and what will happen?

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs [hunk 5]
```diff
@@ -105,6 +113,29 @@ private void Init()
...
+        }
+
+        private void InitSize()
+        {
+            this.Window.Width = 1200 * sizeW;
+            this.Window.Height = 700 * sizeH;
+            this.Window.StylesPreviewGrid.Width = 560 * sizeW;
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs]: these can be extracted to become constants

File: PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs [hunk 6]
```diff
@@ -88,6 +95,7 @@ private void Init()
         {
             try
             {
+                InitSizePosition();
```
Comment (kai33) [inline PowerPointLabs/PowerPointLabs/PictureSlidesLab/View/PictureSlidesLabWindow.xaml.cs]: move this call before line `Logger.Log("PSL begins");`, and you will feel the effect better.
reason: see `SetTimeout(Init, 800);`

I'm fine with the rest of codes
